tests/e2e/e2e_certmanager_test.go

```diff
@@ -0,0 +1,110 @@
+package e2e
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("1 Client pod -> 1 Server pod test using cert-manager", func() {
+	Context("CertManagerSimpleClientServer", func() {
+		const sourceNs = "client"
+		const destNs = "server"
+		var ns []string = []string{sourceNs, destNs}
+
+		It("Tests HTTP traffic for client pod -> server pod", func() {
+			// Install OSM
+			installOpts := td.GetOSMInstallOpts()
+			installOpts.certManager = "cert-manager"
```

Comment:
I'd suggest to make these an enum at common.go at this point

Reply:
I'd vote to skip that step and make it an enum somewhere in `pkg/` that we can just import here.